docs/content/preview/yugabyte-platform/anywhere-automation/yb-kubernetes-operator.md

```diff
@@ -275,6 +275,44 @@ spec:
   keyspace: <keyspace overide>
 ```
 
+#### Service account for backup
+
+In AWS, you can attach a service account to database pods; the account can then be used to access storage in S3 or GCS. The service account used for the database pods should have annotations for the IAM role. The service account to be used can be applied to the DB pods as helm override with provider/universe level overrides. The IAM role used should be sufficient to access S3.
+
+To enable IAM roles for pods, set the **Use S3 IAM roles attached to DB node for Backup/Restore** Universe Configuration option (config key `yb.backup.s3.use_db_nodes_iam_role_for_backup`) to true. Refer to [Manage runtime configuration settings](../../administer-yugabyte-platform/manage-runtime-config/).
+
+The operator pod( YBA ) should have the IAM role for cloud storage access attached to its service account.
+
+For more information, refer to [Enable IAM roles for service accounts](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) in the AWS documentation, and [Authenticate to Google Cloud APIs from GKE workloads](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) in the Google Cloud documentation.
```

Comment:
Can we separate GCS and S3 ? 
The above runtime flag is only gating S3 support so would be best to have S3 in its subsection and GCS in its own.